diff --git a/node_modules/react-native-graph/src/AnimatedLineGraph.tsx b/node_modules/react-native-graph/src/AnimatedLineGraph.tsx
index 7bba35a..554ffcb 100644
--- a/node_modules/react-native-graph/src/AnimatedLineGraph.tsx
+++ b/node_modules/react-native-graph/src/AnimatedLineGraph.tsx
@@ -12,7 +12,6 @@ import {
   vec,
   Group,
   PathCommand,
-  useSharedValueEffect,
   mix,
   Circle,
   Shadow,
@@ -96,8 +95,8 @@ export function AnimatedLineGraph({
     return isActive.value ? 1 : 0
   }, [])
   const indicatorPulseAnimation = useSharedValue(0)
-  const indicatorPulseRadius = useValue(INDICATOR_PULSE_BLUR_RADIUS_SMALL)
-  const indicatorPulseOpacity = useValue(1)
+  const indicatorPulseRadius = useSharedValue(INDICATOR_PULSE_BLUR_RADIUS_SMALL)
+  const indicatorPulseOpacity = useSharedValue(1)
 
   const positions = useComputedValue(
     () => [
@@ -445,22 +444,18 @@ export function AnimatedLineGraph({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [indicatorPulsating])
 
-  useSharedValueEffect(
-    () => {
-      if (pulseTrigger.value === 0) {
-        indicatorPulseRadius.current = mix(
-          indicatorPulseAnimation.value,
-          INDICATOR_PULSE_BLUR_RADIUS_SMALL,
-          INDICATOR_PULSE_BLUR_RADIUS_BIG
-        )
-        indicatorPulseOpacity.current = mix(indicatorPulseAnimation.value, 1, 0)
-      } else {
-        indicatorPulseRadius.current = 0
-      }
-    },
-    indicatorPulseAnimation,
-    pulseTrigger
-  )
+  useEffect(() => {
+    if (pulseTrigger.value === 0) {
+      indicatorPulseRadius.value = mix(
+        indicatorPulseAnimation.value,
+        INDICATOR_PULSE_BLUR_RADIUS_SMALL,
+        INDICATOR_PULSE_BLUR_RADIUS_BIG
+      )
+      indicatorPulseOpacity.value = mix(indicatorPulseAnimation.value, 1, 0)
+    } else {
+      indicatorPulseRadius.value = 0
+    }
+  }, [indicatorPulseAnimation, pulseTrigger])
 
   const axisLabelContainerStyle = {
     paddingTop: TopAxisLabel != null ? 20 : 0,
